### ISSUE TITLE
example response not working due to KeyError

### ISSUE THREAD

**Issue by codeasashu** [zalando/connexion#1078]
### Description
I am using `operation.example_response()` in my custom resolver. I am getting `None` as my example, even though the example exists for given status code and content type. 

### Expected behaviour
`operation.example_response()` should return valid example when exists for status code and content type.

### Actual behaviour
`operation.example_response()` is returning None when exists for status code and content type.

### Steps to reproduce
Using [hello world yaml](https://github.com/zalando/connexion/blob/master/examples/openapi3/helloworld/openapi/helloworld-api.yaml) 

```python
import connexion
from connexion.mock import MockResolver

class CustomResolver(MockResolver):
    def mock_operation(self, operation, *args, **kwargs):
        response, code = operation.example_response() # Returns None, 200

app = connexion.FlaskApp(__name__, specification_dir='swagger/')
app.add_api('helloworld-api.yaml', resolver=CustomResolver(True))
app.run(port=8080)
```
When I do `http post  :8080/getting/dave` I get 
```sh
HTTP/1.0 200 OK
Content-Length: 0
Content-Type: text/plain; charset=utf-8
Date: Sat, 23 Nov 2019 09:44:16 GMT
Server: Werkzeug/0.16.0 Python/3.7.4

```

### Additional info:

Output of the commands:

- `python --version`
Python 3.7.4

- `pip show connexion | grep "^Version\:"`
Version: 2.4.0

**Comment by solson-infoarmor** [comment]
When will this be released on PyPi? The mock functionality is completely broken without the fix. 

My quick and dirty was to change utils.py deep_get to:

`    if not keys:
        return obj
    return deep_get(obj[keys[0]], keys[1:])`

**Comment by codeasashu** [comment]
@solson-infoarmor This has been fixed in #1088

**Comment by dtkav** [comment]
Hey @solson-infoarmor - there's not a fixed release cadence, and it depends on the availability of folks at zalando.
pip can pull from this git repo though, so you should be able to point to the head of master in the meantime.
```
pip install git+https://github.com/zalando/connexion.git@3446508c3d7545ff9b50704a3d3a4034390fa568
```
Helping to test the stability of master is one way to help.
